plugins/inputs/aos/aos_test.go

```diff
@@ -25,7 +25,7 @@ func TestExtractProbeMessage(t *testing.T) {
 	plugin := &Aos{
 		Port:            PORT,
 		Address:         "127.0.0.1",
-		StreamingType:   []string{"alerts"},
+		StreamingType:   []string{"perfmon"},
 		AosServer:       "127.0.0.1",
```

Comment:
streaming type changed in 4.0.0?

Reply:
No - even in 3.2.0 the probe_message was part of PerfMon so I changed it to reflect the actual message type.
```
message PerfMon {
  oneof data {
     InterfaceCounters interface_counters = 1;
     SysResourceCounters system_resource_counters = 2;
     GenericPerfmonMessage generic = 4;
     ProbeMessage probe_message = 5;
  }
  optional float time_delta = 3;
}
```